tools/pythonpkg/README.md

```diff
@@ -0,0 +1,37 @@
+This is the DuckDB Python package
+
+## Installing locally
+
+Set the prefix path:
+
+    export DUCKDB_PREFIX=/path/to/install/duckdb
+
+### Setup for development
+
+Install the package from the root of the DuckDB reposity:
+
+    cd ../..
+    pip3 install --prefix $DUCKDB_PREFIX -e tools/pythonpkg
+
+This creates a package that uses the files in `tools/pythonpkg`, the
+best option during development.
+
+### Setup for cloud storage
+
+Alternatively, you may need the package files to reside under the same
+prefix where the library is installed; e.g., when installing to cloud
+storage from a notebook.
+
+First, get the version number from the repository.
+
+    cd tools/pythonpkg
+    export DUCKDB_VERSION=$(python setup.py --version)
```

Comment:
You can use `SETUPTOOLS_SCM_PRETEND_VERSION` here instead, this will automatically be picked up by `setuptools_scm` without the need for `sed`.
```suggestion
    export SETUPTOOLS_SCM_PRETEND_VERSION=$(python setup.py --version)
```